index.html

```diff
@@ -42,6 +42,14 @@ <h1>OnGrid FullStack Bootcamp</h1>
      <td>
       Varlamov
      </td>
+    </tr>
+       <tr>
```

Comment:
unneccessary space breaks padding structure and makes the document less readable